tests/requirements.yml

```diff
@@ -1,6 +1,6 @@
 - src: https://github.com/idealista-tech/java-role
```

Comment:
We should use https://github.com/idealista/... instead of https://github.com/idealista-tech/... :)